chat_shell/chat_shell/tools/knowledge_factory.py

```
@@ -207,3 +251,57 @@ async def get_knowledge_base_meta_list(
+                if item.get("rag_enabled", False):
+                    logger.debug(
+                        f"[knowledge_factory] KB {item.get('id')} has RAG enabled"
+                    )
+                    return True
+
+            logger.info(
+                f"[knowledge_factory] No KB has RAG enabled among {knowledge_base_ids}"
+            )
+            return False
+
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)